src/meson.build

```diff
@@ -29,18 +29,18 @@ if get_option('gusb')
   client_dep += gusb
 endif
 
-if get_option('libarchive')
+if libarchive.found()
   daemon_dep += libarchive
 endif
-if get_option('sqlite')
+if sqlite.found()
   daemon_dep += sqlite
   client_dep += sqlite
 endif
```

Comment:
These can be simplified to unconditionally add them as dependencies, as long as they are defined in the global scope there is no downside to adding a not-found-dependency to the list of deps.

Reply:
Awesome, will adjust it.